src/app.ts

```diff
@@ -1,6 +1,87 @@
 
 const startApp = async () => {
+    const tbody = document.querySelector('[data-sink]')
+    const tr = tbody?.querySelectorAll('tr')
```

Comment:
Codacy has a fix for the issue: [Replace `····const·tr·=·tbody?.querySelectorAll('tr')` with `··const·tr·=·tbody?.querySelectorAll("tr");`](https://app.codacy.com/gh/alic-xc/Test/pullRequest?prid=9965044)
```suggestion
  const tr = tbody?.querySelectorAll("tr");
```